docs/api/registration-base-url-resource.md

```diff
@@ -166,7 +166,8 @@ dependencyGroups         | array of objects           | no       | The dependenc
 description              | string                     | no       | 
 iconUrl                  | string                     | no       | 
 id                       | string                     | yes      | The ID of the package
-licenseUrl               | string                     | no       | 
+licenseUrl               | string                     | no       |
+licenseExpression        | string                     | no       | The license expression specified in the package manifest.
```

Comment:
`manifest.` -> `manifest`

Reply:
I don't think it's necessary to make this note. None of the other fields have it and it's kind of an implementation detail about where it came from. The point is, the server considers this package to be licensed using that SPDX expression.